gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -18106,7 +18208,11 @@ properties:
     enum:
```

Comment:
I'm not sure if I'm missing something here, but  I don't see the "supratentorial_localization" property
--